Assets/Mirror/Runtime/Messages.cs

```diff
@@ -153,11 +154,25 @@ public class DisconnectMessage : EmptyMessage {}
 
     public class ConnectMessage : EmptyMessage {}
 
-    public class SceneMessage : StringMessage
+    public class SceneMessage : MessageBase
     {
-        public SceneMessage(string value) : base(value) {}
+        public string sceneName;
+        public LoadSceneMode sceneMode; // Single = 0, Additive = 1
+        public LocalPhysicsMode physicsMode; // None = 0, Physics3D = 1, Physics2D = 2
 
-        public SceneMessage() {}
+        public override void Deserialize(NetworkReader reader)
+        {
+            sceneName = reader.ReadString();
+            sceneMode = (LoadSceneMode)reader.ReadInt32();
+            physicsMode = (LocalPhysicsMode)reader.ReadInt32();
```

Comment:
please use ReadByte + Write(byte) for sceneMode and physicsMode if they all range between 0 and 3 anyway

Reply:
requested change made. confirmed working with additive scene example.